docs/openapi/api.yaml

```diff
@@ -99,6 +101,8 @@ paths:
     $ref: './audit-api.yaml#/latest-audit-for-site'
   /sites/{siteId}/{auditType}:
     $ref: './audit-api.yaml#/update-handler-type-config-for-site'
+  /sites/audits:
```

Comment:
this endpoint essentially updates the sites field inside the audit configuration. I think the endpoint should exist inside the configuration controller and should have a URL such as:

```
POST /configurations/sites/{siteId}/audits/{auditType}
{ enabled: true }
```

Reply:
For the HTTP API, however, I’d prefer to keep the bulk update functionality.

Currently, users have a straightforward way to enable audits for individual sites, but, as practice shows, supporting bulk updates (despite minor deviations from REST) helps lay the foundation for future system modifications, including potential performance improvements.
Why do we need single updates if we can use bulk updates with the same simplicity for the end user? :)

So, I adjusted the request structure, but it remains in bulk format.
```
[
      { baseURL: 'https://site1.com', auditType: 'cwv', enable: true },
      { baseURL: 'https://site1.com', auditType: '404', enable: true },
      { baseURL: 'https://site2.com', auditType: 'cwv', enable: false },
      { baseURL: 'https://site2.com', auditType: '404', enable: false },
]
```
Also, I’m using the PATCH method since we’re not overriding the configuration entirely but rather updating it incrementally.

Unless, of course, you have critical objections.